extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/telemetry/TelemetryConstants.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.websockets.next.runtime.telemetry;
+
+public interface TelemetryConstants {
```

Comment:
I think that for constants it's usually better to define a `final` class with private constructor.